### ISSUE TITLE
Move cursor to $a after adding a new field

### ISSUE THREAD

**Issue by viola-v** [dag-hammarskjold-library/dlx-rest#757]
Currently, when I create a new field and click on the Tab or enter key, the default subfields are added. but my cursor is still stuck in the field. I think it would be nice if the cursor went straight to subfield $a, so that I can start entering data without having to tab through the indicators first. Most fields don't use indicators, so this would save time and clicking, and for those that do, we can go back and add them.

**Comment by jbukhari** [comment]
Yes, an extra necessary keypress was added when the default subfield mechanism was added. The standard behavior should be retained (i.e., the behavior when editing a tag of any other field).

Also related to open PR #722 . There, we have two possible solutions implemented for skipping from the tag field to the first subfield,  but we never returned to resolve the issue. One solution is to alter which fields are "tabable", so that the tab key takes you straight from the tag field to the first subfield value field (the fields in between therefore are not accessible by tabbing). The other solution is to leave tab behavoiour as is, but go straight to the subfield value on pressing the enter key from the tag field.

**Comment by viola-v** [comment]
I think we had decided on this solution: "The other solution is to leave tab behavior as is, but go straight to the subfield value on pressing the enter key from the tag field." Can this be implemented before UAT? It would make a big difference for usability.
